frontend/src/scenes/organization/Settings/membersLogic.tsx

```diff
@@ -15,37 +15,31 @@ export const membersLogic = kea<membersLogicType>({
             member,
             level,
         }),
-        postRemoveMember: (memberUuid: string) => ({ memberUuid }),
+        postRemoveMember: (userUuid: string) => ({ userUuid }),
```

Comment:
I think for consistency this should just be called `userId` in the frontend

Reply:
I'd definitely like to stick to calling it `userUuid` when referring to `user.uuid` and `userId` when `user.id`. They're very different, and importantly it's `user.id` that's still the primary key.

Reply:
But in the frontend, the user ID (int pk) does not exist in any context, or am I mistaken? As in, for all that frontend is concerned, only the ID exists.

Reply:
Pretty much isn't used, but it still exists on the model, so context-switching would become a bit weird (and potentially error-prone) having to remember that `id` means a whole different field in the frontend than it means in the backend.